src/transaction/log_applier.c

```diff
@@ -7262,12 +7296,12 @@ la_log_page_check (const char *database_name, const char *log_path, INT64 page_n
       free_and_init (la_Info.log_data);
     }				/* check_copied_info */
 
-check_copied_info_end:
+check_copied_log_page_info_end:
   if (error != NO_ERROR)
     {
       printf ("%s\n", db_error_string (3));
     }
-
+  /*The -p option does not affect any results in the applyinfo utility. So, here returns the NO_ERROR */
```

Comment:
```suggestion

  /* The -p option does not affect any results in the applyinfo utility. So, here returns the NO_ERROR */
```

Reply:
Ok, I will apply it both of them. Thank you